code/mission/missionparse.cpp

```diff
@@ -4574,6 +4574,14 @@ void parse_wing(mission *pm)
 	// wing could be created if it needs to be
 	wingp->wave_delay_timestamp = timestamp(0);
 
+	// Use a custom formation if specified
+	if (optional_string("+Formation Index:")) {
+		stuff_int(&(wingp->formation));
+	}
+	else { // else use the retail formation
+		wingp->formation = -1;
```

Comment:
This should default to the species default formation (which itself defaults to -1).

Reply:
Also, add a corresponding block in `missionsave.cpp` (for both FRED and qtFRED) to save the formation index.  Look at how `wing_squad_filename` is saved for an example.